src/simulate.py

```diff
@@ -341,9 +349,46 @@ def simulate(self, computation_dfg, hw):
                         * hw.latency[node_data["function"]] # ns
                     )
 
+                # wires
+                active_net_len = len(active_net)
+                computation_nodes = list(computation_dfg.out_edges(node))
```

Comment:
don't call this computation nodes call it something more meaningful like out_nodes